samples/nrf_rpc/protocols_serialization/server/sysbuild.conf

```diff
@@ -0,0 +1,4 @@
+SB_CONFIG_PARTITION_MANAGER=n
```

Comment:
Also, this could probably be enabled with a snippet and its `SB_EXTRA_CONF_FILE` argument. Alternatively, you could use `FILE_SUFFIX=mcuboot` and rename this to `sysbuild_mcuboot.conf`.